app/components/interview_bookings_item_component.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class InterviewBookingsItemComponent < ViewComponent::Base
+  attr_accessor :interview
+
+  def initialize(interview)
+    @interview = interview
+  end
+
+  def formatted_time(interview)
+    interview.date_and_time.to_s(:govuk_date_and_time).squish
+  end
+
+  def formatted_date(interview)
+    interview.date_and_time.to_s(:govuk_date).squish
+  end
+
+  def provider_name(interview)
+    interview.provider.name
+  end
+
+  def location(interview)
+    safely_format_with_hyperlinks(interview.location)
+  end
+
+  def additional_details(interview)
+    safely_format_with_hyperlinks(interview.additional_details)
+  end
+
+  def in_the_past?
+    interview.date_and_time + 1.hour < Time.zone.now
+  end
```

Comment:
Thanks for adding the 1 hour margin! Might help out some candidates who are running late for the interview who wouldn't otherwise be about to see the interview details...!

Should we add a comment to explain this? And possibly we could even bump it up to 2 hours, for anyone who’s _really_ late?

Reply:
@frankieroberto would it make sense to just compare by date rather than adding an arbitrary buffer, so that the details don't disappear until midnight?